doc/manual/variables-and-scoping.rst

```diff
@@ -159,33 +184,37 @@ keyword ``global``::
 
 The location of both the ``local`` and ``global`` keywords within the
 scope block is irrelevant.  The following is equivalent to the last
-example (although stylistically worse)::
+example (although stylistically worse):
+
+.. doctest::
 
-    for i=1:10
-        z = i
-        global z
-    end
+    julia> for i=1:10
+               z = i
+               global z
+           end
 
     julia> z
     10
 
 Multiple global or local definitions can be on one line and can also
-be paired with assignments::
+be paired with assignments:
 
-    for i=1:10
-        global x=i, y, z
-        local a=4, b , c=1
-    end
+.. doctest::
+
+    julia> for i=1:10
+               global x=i, y, z
+               local a=4, b , c=1
+           end
 
 
 .. _man-soft-scope:
 
 Soft Local Scope
 ^^^^^^^^^^^^^^^^
 
-  In a soft local scope, all variables are inherited from its parent
-  scope unless a variable is specifically marked with the keyword
-  ``local``.
+In a soft local scope, all variables are inherited from its parent
+scope unless a variable is specifically marked with the keyword
+``local``.
```

Comment:
I believe that this indent was intentional &ndash; to give it some additional emphasis probably? (@mauro3 wrote it originally I think).